cluster/images/hyperkube/Makefile

```diff
@@ -1,13 +1,27 @@
 # build the hyperkube image.
 
-VERSION=v1.1.1
+VERSION=v1.1.2
+ARCH=amd64
+BASEIMAGE=debian:jessie
+
+## Comment in for arm builds, must be run on an arm machine
+# ARCH=arm
+# need to escape '/' for the regexp below
+# BASEIMAGE=armbuild\\/debian:jessie
 
 all:
 	cp ../../saltbase/salt/helpers/safe_format_and_mount .
 	cp ../../saltbase/salt/generate-cert/make-ca-cert.sh  .
-	curl -O https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/amd64/hyperkube
+	curl -O https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/${ARCH}/hyperkube
 	sed -i "s/VERSION/${VERSION}/g" master-multi.json master.json
-	docker build -t gcr.io/google_containers/hyperkube:${VERSION} .
-	gcloud docker push gcr.io/google_containers/hyperkube:${VERSION}
+        sed -i "s/ARCH/${ARCH}/g" master-multi.json master.json
```

Comment:
there's an indentation issue here (offset by one space) but you can fix it in a followup PR